src/web-animations-next-animation.js

```diff
@@ -70,11 +98,12 @@
       if (!this.effect || !this._isGroup)
         return;
       var offset = this.effect._timing.delay;
+      this._removeChildren();
```

Comment:
What does _removeChildren actually do? Should we rename these as constructChildAnimations and removeChildAnimations?